greenpithumb/pi_io.py

```diff
@@ -0,0 +1,43 @@
+#import RPi.GPIO as GPIO
+
+# TODO(mtlynch): Move these pin constants to a configuration file so they're not
+# hardcoded in.
+
+# Raspberry Pi pin to which each component is connected
+PIN_PUMP = 0
```

Comment:
(kind of out of scope, but since we're touching it): I think we should take this out of here and have the caller to `Pump` specify the pump's pin in the constructor. It would match all the ADC components where the caller has to specify the ADC channel in the constructor.